This patch sets the default location for the GeSHi library
to match FreeBSD's GeSHi port layout.

--- geshifilter.inc.orig	2009-03-21 14:40:30.000000000 +0300
+++ geshifilter.inc	2009-03-21 14:40:54.000000000 +0300
@@ -7,7 +7,7 @@
  */
 
 function _geshifilter_get_geshi_dir() {
-  return variable_get('geshifilter_geshi_dir', drupal_get_path('module', 'geshifilter') .'/geshi');
+  return variable_get('geshifilter_geshi_dir', '%%LOCALBASE%%/lib/php/geshi');
 }
 
 /**
